### DISCUSSION TITLE
Disable adding new blocks?

### DISCUSSION THREAD

**Discussion by userbig** [codex-team/editor.js#1925] [Q&A]
I want to know if there any options to disable adding new blocks to editor body? 
Thanks

**Comment by Dorianux** [comment]
I have the same question.

It would be very usefull to be able to only edit predefined content parts (that are provided as existing values).
The user gets the blocks that can be edited in the according place and should not be able to add any additonal blocks.
This way editorjs could be used as editing tool for semi-structured data.

**Comment by nake89** [comment]
I am also interested in this one. If anyone has a workaround or a plugin that could achieve this, I would be very interested.

**Comment by kurucu** [comment]
Has anyone worked this out? I thought about a workaround being to trap and cancel an event fired by the editor before a new block is assed, but haven't worked out whether that's possible.

**Comment by nenad-coniq** [comment]
This is the workaround solution I have: [https://github.com/codex-team/editor.js/discussions/1974#discussioncomment-2196059](https://github.com/codex-team/editor.js/discussions/1974#discussioncomment-2196059)

P.S. I care only for specific tools i.e. toggling on and off display styling directly on toolbox menu.
